watertap/costing/unit_models/heat_exchanger.py

```diff
@@ -28,12 +28,20 @@ def build_heat_exchanger_cost_param_block(blk):
         units=pyo.units.dimensionless,
     )
 
+    blk.steam_cost = pyo.Var(
+        initialize=0.008,
+        units=pyo.units.USD_2018 / (pyo.units.kg),
+        doc="steam cost per kg",
+    )
```

Comment:
It would be nice to have this update reflected in the documentation, but if it would hold up this PR, we can just update the documentation later (make an issue?)